parmed/symmetry.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+class Symmetry(object):
+    def __init__(self, lines=None):
```

Comment:
Converting PDB lines into a `Symmetry` object should be done as a method in the PDB parsing class.  Currently, making a `Symmetry` object from anything other than a set of PDB lines following a REMARK 290 is awkward.  `Symmetry` should have a format-agnostic API.

It should either take the full tensor as an array (and store it as a numpy array) or something else that feels natural to people that regularly work with symmetry operators.

You should not be able to instantiate an "empty" `Symmetry` object -- if a format doesn't define it, then `structure.symmetry` is left as `None` and this constructor will never be called.